debug/trace/memory/memory.go

```diff
@@ -85,6 +85,6 @@ func NewTracer(opts ...trace.Option) trace.Tracer {
 	return &Tracer{
 		opts: options,
 		// the last 64 requests
-		buffer: ring.New(64),
+		buffer: ring.New(640),
```

Comment:
This might be a bit controversial.

Reply:
That is definitely controversial. Max I'd do is 256